test/unit/specs/components/DecisioningEngine/createContextProvider.spec.js

```diff
@@ -11,7 +11,8 @@ governing permissions and limitations under the License.
 */
 import createContextProvider from "../../../../../src/components/DecisioningEngine/createContextProvider";
 import createEventRegistry from "../../../../../src/components/DecisioningEngine/createEventRegistry";
-
+import parseUrl from "../../../../../src/utils/parseUrl";
+// TODO: Need to mock url using some library like sinon, also write comprehensive tests for all the scenarios
```

Comment:
In the meantime, we do have [Jasmine Spies](https://jasmine.github.io/api/edge/Spy.html).